CHANGELOG.rst

```diff
@@ -31,6 +31,11 @@ Fixed
   (bug fix)
 
   Contributed by carbineneutral. #3534 #3544
+* st2 pack commands now work when StackStorm servers are behind a HTTP/HTTPS proxy. You can set
+  ``http_proxy`` or ``https_proxy`` environment variables for ``st2api`` and ``st2actionrunner``
```

Comment:
``HTTP_PROXY`` and `HTTPS_PROXY`` should probably be upper case to follow good practices and  conventions.